package.json

```diff
@@ -7,13 +7,15 @@
   "private": true,
   "dependencies": {
     "@types/highlight.js": "9.12.3",
+    "@types/redux-mock-store": "1.0.1",
     "@typescript-eslint/parser": "2.6.0",
     "bootstrap": "3.4.1",
     "bootstrap-colorpicker": "2.5.2",
     "chart.js": "2.9.2",
     "compass-mixins": "0.12.10",
     "core-js": "3.3.6",
     "css-vars-ponyfill": "2.1.2",
+    "dom-to-react": "1.0.0",
```

Comment:
What's `dom-to-react` needed for?

Reply:
For some reason, the compiler was complaining about this at certain moment. Maybe I had some change at the moment that made this requirement. I just removed it from `package.json` and seems to be not complaining anymore.